src/Chunk.cpp

```diff
@@ -1885,7 +1885,7 @@ bool cChunk::AddClient(cClientHandle * a_Client)
 
 void cChunk::RemoveClient(cClientHandle * a_Client)
 {
-	std::remove(m_LoadedByClient.begin(), m_LoadedByClient.end(), a_Client);
+	m_LoadedByClient.erase(std::remove(m_LoadedByClient.begin(), m_LoadedByClient.end(), a_Client));
```

Comment:
And if there are two copies of a pointer in that list then we've got bigger problems. But I'll create another PR that asserts if we need to delete more than one item.